api/v1alpha1/compression_types.go

```diff
@@ -0,0 +1,21 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package v1alpha1
+
+type GzipCompressor struct {
+}
+
+type Compression struct {
```

Comment:
can we add a doc string around defaults ? enabled for responses and disabled for requests as well as per compression type defaults

Reply:
yup ,will do, thanks